internal/identity/identitymanager.go

```diff
@@ -33,36 +34,44 @@ import (
 )
 
 type Manager interface {
-	ResolveInputIdentity(ctx context.Context, identity *fftypes.Identity) (err error)
-	ResolveSigningKey(ctx context.Context, inputKey string) (outputKey string, err error)
-	ResolveSigningKeyIdentity(ctx context.Context, signingKey string) (author string, err error)
-	ResolveLocalOrgDID(ctx context.Context) (localOrgDID string, err error)
-	GetLocalOrgKey(ctx context.Context) (string, error)
-	OrgDID(org *fftypes.Organization) string
-	GetLocalOrganization(ctx context.Context) (*fftypes.Organization, error)
+	ResolveInputSigningIdentity(ctx context.Context, namespace string, msgSignerRef *fftypes.SignerRef) (err error)
+	ResolveInputSigningKeyOnly(ctx context.Context, namespace string, resolveViaBlockchainPlugin bool) (signingKey string, err error)
```

Comment:
I think the var names here are wrong - second param is the key, not the namespace, right?

I would also say we can drop the "Only" and call it `ResolveInputSigningKey`, but maybe that's just me.

Reply:
Yeah, really fought with the function name spelling here. The key (no pun intended) thing about the name here is to state it's the unusual case. Most code paths when resolving a signing key, you want to resolve it in FireFly with an associated identity. In a couple of cases it want's to perform the special case of just "resolving" a key via the blockchain plugin, without resolving what identity might/might-not be associated with it.

Reply:
Resolved alongside the `normalization` rename